### ISSUE TITLE
Allow for better formatting of curried parameter lists

### ISSUE THREAD

**Issue by sschaef** [scala-ide/scalariform#159]
_From @nairbv on January 1, 2016 17:46_

As described in a comment on:
https://github.com/daniel-trinh/scalariform/issues/36

To be able to use multiple parameter lists in a nicely formatted way, while keeping a strict line-length restriction in scalastyle, it would be nice to be able to preserve new lines and indentation between parameter lists.

E.g. I would like to be able to write code like:

```
def aReallyLongMethodName(thatHasSomeParameters:WithLongTypes)
                         (andThenAnother:ParameterList) = foo
```

Without having it reformatted to:

```
def aReallyLongMethodName(thatHasSomeParameters:WithLongTypes)(andThenAnother:ParameterList) = foo
```

The simplest solution is just to preserve any existing whitespace between ) and (, and I think I could submit a pull request to do that.

There might be a nicer way to achieve the same result, e.g. actually formatting the indentation of the next parameter list to match the first, but that might take more effort to implement and might reduce flexibility for the programmer to indent to a specific level.

_Copied from original issue: daniel-trinh/scalariform#104_

**Comment by sschaef** [comment]
_From @nairbv on January 1, 2016 17:59_

ah, I didn't see: https://github.com/daniel-trinh/scalariform/pull/4

**Comment by godenji** [comment]
Fixed in 0.2.4
